docs/maui/layouts/UniformItemsLayout.md

```diff
@@ -0,0 +1,113 @@
+---
+title: UniformItemsLayout - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The UniformItemsLayout is a layout where all rows and columns have the same size.
+ms.date: 04/12/2022
+---
+
+# UniformItemsLayout
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `UniformItemsLayout` is a layout that organizes its children in rows and columns. Each child view will have the same width and height.
+
+## Building a UniformItemsLayout
+
+A `UniformItemsLayout` can be created in XAML or C#:
+
+### XAML
+
+```xml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyProject.MyContentPage">
+
+    <toolkit:UniformItemsLayout>
+        <BoxView BackgroundColor="Blue" HeightRequest="25" WidthRequest="25"/>
+        <BoxView BackgroundColor="Yellow" HeightRequest="25" WidthRequest="25"/>
+        <BoxView BackgroundColor="Red" HeightRequest="25" WidthRequest="25"/>
+        <BoxView BackgroundColor="Black" HeightRequest="25" WidthRequest="25"/>
+    </toolkit:UniformItemsLayout>
+    
+</ContentPage>
+```
+
+### C#
+
+```csharp
+using CommunityToolkit.Maui.Views;
+
+var page = new ContentPage
+{
+    Content = new UniformItemsLayout
+    {
+        Children = 
+        {
+            new BoxView { HeightRequest = 25, WidthRequest = 25, BackgroundColor = Colors.Blue },
+            new BoxView { HeightRequest = 25, WidthRequest = 25, BackgroundColor = Colors.Yellow },
+            new BoxView { HeightRequest = 25, WidthRequest = 25, BackgroundColor = Colors.Red },
+            new BoxView { HeightRequest = 25, WidthRequest = 25, BackgroundColor = Colors.Black }
+        }
+    }
+};
+```
+
+## Customizing a UniformItemsLayout
+
+A `UniformItemsLayout` allows the number of columns and rows to be limited through the use of the `MaxRows` and `MaxColumns` properties:
```

Comment:
```suggestion
A `UniformItemsLayout` allows the number of columns and rows to be limited through the use of the `MaxRows` and `MaxColumns` properties.

In the below example we limit the `MaxColumns` and `MaxRows` to 1, this means that only one `BoxView` will be visible on the screen. In this scenario it is the last item added that will be visible. So in the example below it will be the black `BoxView`.
```